sdk/digitaltwins/Azure.DigitalTwins.Core/src/Queries/QueryBuilders/Linq/Evaluator.cs

```diff
@@ -0,0 +1,84 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Linq.Expressions;
+using System.Diagnostics;
+
+namespace Azure.DigitalTwins.Core.QueryBuilder
+{
+    internal static class Evaluator
```

Comment:
All classes could use doc comments for future code owners to understand **what** a class represents and **what** it does.

Reply:
I'd also recommend adding a doc comment that these were borrowed from https://github.com/Azure/azure-sdk-for-net/tree/main/sdk/tables/Azure.Data.Tables/src/Queryable so it's easier to come bug my team if anyone ever has questions about these classes. :smile: